packages/cli-kit/src/public/node/error-handler.ts

```diff
@@ -121,11 +122,13 @@ export async function sendErrorToBugsnag(
 
     if (report) {
       initializeBugsnag()
+      const userId = await getLastSeenUserIdAfterAuth()
```

Comment:
This is returning `unknown` when not found. For example, when running `shopify version` unauthenticated.

@MitchDickinson should we better send `undefined` in that case so Observe sets the IP in that case? Not sure what's best...